helm/values.yaml

```diff
@@ -1,28 +1,30 @@
 environment: development
+name: mine-support-user-service
+image: mine-support-user-service
 postgresUsername: postgres@mine-support2
 postgresPassword: changeme
-containers:
-  service:
-    name: mine-support-user-service
-    port: 3002
-    registry:
-    image: mine-support-user-service
-    tag: latest
-    imagePullPolicy: IfNotPresent
-    requestMemory: 30Mi
-    requestCpu: 10m
-    limitMemory: 60Mi
-    limitCpu: 40m
-    runAsNonRoot: true
-    runAsUser: 1000
-    readOnlyRootFilesystem: true
-    allowPrivilegeEscalation: false
-    restartPolicy: Always
-database:
-  name: mine-support-postgres-users
-  externalName: host.docker.internal
-  port: 5432
-replicas: 1
+postgresDatabase: mine_users
+postgresHost: mine-support-postgres-users
+postgresExternalName: host.docker.internal
+postgresPort: 5432
```

Comment:
the postgres settings are shared by the user service, and the external database service so have been moved up to the top level. Changes were also made to the code to ensure the hard coded values were being set the same as in the Helm chart.